hack/ci-e2e.sh

```diff
@@ -11,6 +11,9 @@
 set -eux
 
 REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")
+BMO_IMAGE_NAME="quay.io/metal3-io/baremetal-operator"
+
+export E2E_CONF_FILE="${REPO_ROOT}/test/e2e/config/ironic.yaml"
```

Comment:
is this needed? Can't we take it away?

Reply:
Yes. Initially it wasn't needed because we have the default in Makefile, but now we also use this env var in the `hack/e2e/ensure_kustomization_paths.sh`, it should be explicit.